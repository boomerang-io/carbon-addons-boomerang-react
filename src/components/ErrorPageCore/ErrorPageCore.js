import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';
import { settings } from 'carbon-components';
import GenericErrorBackground from './GenericErrorBackground';
const { prefix } = settings;

ErrorPageCore.propTypes = {
  className: PropTypes.string,
  graphic: PropTypes.node,
  header: PropTypes.node,
  message: PropTypes.node,
  style: PropTypes.object,
  title: PropTypes.node,
};

export default function ErrorPageCore({ className, graphic, header, message, style, title }) {
  console.log();
  return (
    <div className={cx(`${prefix}--bmrg-error-page-core`, className)} style={style}>
      {graphic ?? <GenericErrorBackground className={`${prefix}--bmrg-error-page-core__background`} />}
      <div className={`${prefix}--bmrg-error-page-core__content`}>
        {header && <header className={`${prefix}--bmrg-error-page-core__header`}>{header}</header>}
        {title && <h1 className={`${prefix}--bmrg-error-page-core__title`}>{title}</h1>}
        {message && <section className={`${prefix}--bmrg-error-page-core__message`}>{message}</section>}
      </div>   
          
    </div>
  );
}
