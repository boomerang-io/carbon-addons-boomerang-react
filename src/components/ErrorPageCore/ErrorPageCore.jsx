import React from "react";
import PropTypes from "prop-types";
import cx from "classnames";
import { prefix } from "../../internal/settings";
import GenericErrorBackground from "./GenericErrorBackground";


ErrorPageCore.propTypes = {
  className: PropTypes.string,
  graphic: PropTypes.node,
  header: PropTypes.node,
  message: PropTypes.node,
  statusUrl: PropTypes.string,
  style: PropTypes.object,
  title: PropTypes.node,
};

ErrorPageCore.defaultProps = {
  header: "Oops!",
  title: "Something looks off, but we're getting a handle of it.",
};

export default function ErrorPageCore({ className, graphic, header, message, statusUrl, style, title }) {
  return (
    <div className={cx(`${prefix}--bmrg-error-page-core`, className)} style={style}>
      {graphic ?? <GenericErrorBackground className={`${prefix}--bmrg-error-page-core__background`} />}
      <div className={`${prefix}--bmrg-error-page-core__content`}>
        {header && <header className={`${prefix}--bmrg-error-page-core__header`}>{header}</header>}
        {title && <h1 className={`${prefix}--bmrg-error-page-core__title`}>{title}</h1>}
        <section className={`${prefix}--bmrg-error-page-core__message`}>
          {message ? (
            message
          ) : (
            <p>
              Hit the back button to return to your previous page, or if you keep finding yourself here,
              <a href={statusUrl}>{` navigate to status `}</a>
              for help.
            </p>
          )}
        </section>
      </div>
    </div>
  );
}
