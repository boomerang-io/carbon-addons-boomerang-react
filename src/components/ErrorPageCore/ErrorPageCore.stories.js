import React from 'react';
import { text } from '@storybook/addon-knobs';

import ErrorPageCoreComponent from './index';

export default {
  title: 'ErrorPageCore',
};

export const Default = () => {
  return (
    <ErrorPageCoreComponent
      header={text('header', 'Header')}
      title={text('title', 'Title')}
      message={text('message', 'Message')}
    />
  );
};

export const MessageLink = () => {
  return (
    <ErrorPageCoreComponent
      title={text('title', 'Title')}
      message={
        <p>
          Hello there, <a href="https://useboomerang.io">use Boomerang!</a>
        </p>
      }
    />
  );
};

MessageLink.story = {
  name: 'message link',
};
