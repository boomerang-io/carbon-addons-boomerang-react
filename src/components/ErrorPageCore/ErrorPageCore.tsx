import React from "react";
import cx from "classnames";
import { prefix } from "../../internal/settings";
import GenericErrorBackground from "./GenericErrorBackground";

ErrorPageCore.defaultProps = {
  header: "Oops!",
  title: "Something looks off, but we're getting a handle of it.",
};

type OwnProps = {
  className?: string;
  graphic?: React.ReactNode;
  header?: React.ReactNode;
  message?: React.ReactNode;
  statusUrl?: string;
  style?: any;
  title?: React.ReactNode;
};

type Props = OwnProps & typeof ErrorPageCore.defaultProps;

export default function ErrorPageCore({ className, graphic, header, message, statusUrl, style, title }: Props) {
  return (
    <div className={cx(`${prefix}--bmrg-error-page-core`, className)} style={style}>
      {graphic ?? <GenericErrorBackground className={`${prefix}--bmrg-error-page-core__background`} />}
      <div className={`${prefix}--bmrg-error-page-core__content`}>
        {header && <h1 className={`${prefix}--bmrg-error-page-core__header`}>{header}</h1>}
        {title && <p className={`${prefix}--bmrg-error-page-core__title`}>{title}</p>}
        <section className={`${prefix}--bmrg-error-page-core__message`}>
          {message ? (
            message
          ) : (
            <p>
              Hit the back button to return to your previous page, or if you keep finding yourself here,
              <a href={statusUrl}>{` navigate to Support Center `}</a>
              for help.
            </p>
          )}
        </section>
      </div>
    </div>
  );
}
