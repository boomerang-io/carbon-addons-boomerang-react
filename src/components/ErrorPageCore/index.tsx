export { default } from "./ErrorPageCore";
export type { Props } from "./ErrorPageCore";
