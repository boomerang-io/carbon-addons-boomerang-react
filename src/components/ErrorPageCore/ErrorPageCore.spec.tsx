import React from "react";
import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import { axe } from "jest-axe";
import ErrorPageCore from "./ErrorPageCore";

const statusUrl = "https://useboomerang.io";

describe("ErrorPageCore", () => {
  test("snapshot", async () => {
    const { baseElement } = render(<ErrorPageCore statusUrl={statusUrl} />);
    expect(baseElement).toMatchSnapshot();
  });

  test("functional", async () => {
    const { getByText } = render(<ErrorPageCore statusUrl={statusUrl} />);
    expect(getByText("Oops!")).toBeInTheDocument();
    expect(getByText("Something looks off, but we're getting a handle of it.")).toBeInTheDocument();
  });

  test("a11y", async () => {
    const { container } = render(<ErrorPageCore statusUrl={statusUrl} />);
    const results = await axe(container);
    expect(results).toHaveNoViolations();
  });
});
