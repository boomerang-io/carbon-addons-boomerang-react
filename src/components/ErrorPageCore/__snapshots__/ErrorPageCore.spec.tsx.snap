// Vitest Snapshot v1

exports[`ErrorPageCore > snapshot 1`] = `
<body>
  <div>
    <div
      class="cds--bmrg-error-page-core"
    >
      <svg
        class="cds--bmrg-error-page-core__background"
        data-name="Isometric Graphics"
        id="prefix__Isometric_Graphics"
        viewBox="0 0 1440 900"
        xmlns="http://www.w3.org/2000/svg"
      >
        <defs>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="prefix__linear-gradient"
            x1="641.4"
            x2="1395.9"
            y1="681.72"
            y2="681.72"
          >
            <stop
              offset="0"
              stop-color="#c1c7cd"
            />
            <stop
              offset="0.39"
              stop-color="#c2c8cd"
              stop-opacity="0.99"
            />
            <stop
              offset="0.53"
              stop-color="#c4c9cf"
              stop-opacity="0.96"
            />
            <stop
              offset="0.63"
              stop-color="#c7cdd2"
              stop-opacity="0.9"
            />
            <stop
              offset="0.71"
              stop-color="#ccd1d6"
              stop-opacity="0.82"
            />
            <stop
              offset="0.78"
              stop-color="#d3d7db"
              stop-opacity="0.71"
            />
            <stop
              offset="0.84"
              stop-color="#dbdfe2"
              stop-opacity="0.58"
            />
            <stop
              offset="0.9"
              stop-color="#e5e8ea"
              stop-opacity="0.42"
            />
            <stop
              offset="0.95"
              stop-color="#f0f2f3"
              stop-opacity="0.24"
            />
            <stop
              offset="0.99"
              stop-color="#fdfdfd"
              stop-opacity="0.03"
            />
            <stop
              offset="1"
              stop-color="#fff"
              stop-opacity="0"
            />
          </lineargradient>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="prefix__linear-gradient-2"
            x1="-0.32"
            x2="642.04"
            y1="649.36"
            y2="649.36"
          >
            <stop
              offset="0"
              stop-color="#fff"
              stop-opacity="0"
            />
            <stop
              offset="0.01"
              stop-color="#fdfdfd"
              stop-opacity="0.03"
            />
            <stop
              offset="0.05"
              stop-color="#f0f2f3"
              stop-opacity="0.24"
            />
            <stop
              offset="0.1"
              stop-color="#e5e8ea"
              stop-opacity="0.42"
            />
            <stop
              offset="0.16"
              stop-color="#dbdfe2"
              stop-opacity="0.58"
            />
            <stop
              offset="0.22"
              stop-color="#d3d7db"
              stop-opacity="0.71"
            />
            <stop
              offset="0.29"
              stop-color="#ccd1d6"
              stop-opacity="0.82"
            />
            <stop
              offset="0.37"
              stop-color="#c7cdd2"
              stop-opacity="0.9"
            />
            <stop
              offset="0.47"
              stop-color="#c4c9cf"
              stop-opacity="0.96"
            />
            <stop
              offset="0.61"
              stop-color="#c2c8cd"
              stop-opacity="0.99"
            />
            <stop
              offset="1"
              stop-color="#c1c7cd"
            />
          </lineargradient>
          <style>
            .prefix__cls-4{fill:#c1c7cd}.prefix__cls-5{fill:#dde1e6}.prefix__cls-6{fill:#f2f4f8}.prefix__cls-7{fill:#878d96}.prefix__cls-17,.prefix__cls-9{opacity:.25}.prefix__cls-10,.prefix__cls-17{fill:#4d5358}.prefix__cls-12{fill:#33b1ff}.prefix__cls-13{fill:#e5f6ff}.prefix__cls-14{fill:#a2a9b0}.prefix__cls-15{fill:#697077}.prefix__cls-16{fill:#fff}
          </style>
        </defs>
        <path
          d="M641.72 464.06l753.86 435.31"
          fill="none"
          stroke="url(#prefix__linear-gradient)"
          stroke-miterlimit="10"
          stroke-width="1.28"
        />
        <path
          d="M0 834.65l641.72-370.59"
          data-name="Do Not Sign"
          fill="none"
          id="prefix__Do_Not_Sign"
          stroke="url(#prefix__linear-gradient-2)"
          stroke-miterlimit="10"
          stroke-width="1.28"
        />
        <path
          d="M641.72 464.06L641.3.63"
          fill="none"
          stroke="#c1c7cd"
          stroke-miterlimit="10"
          stroke-width="1.28"
        />
        <path
          class="prefix__cls-4"
          d="M403.41 636.56c16.56 9.57 16.63 25.06.19 34.61-8.85 5.15-35.83 14.13-47.46 13.72-10-.34-4.63-9.32-12.27-13.73-16.54-9.55-16.63-25-.19-34.59s43.18-9.57 59.73-.01z"
          id="prefix__Shadow"
        />
        <g
          id="prefix__Pot"
        >
          <path
            class="prefix__cls-5"
            d="M395.16 607.14l-.68 54c0 6.17-4.08 12.33-12.18 17-16.31 9.48-42.81 9.48-59.24 0-8.26-4.77-12.39-11-12.37-17.27l.15-52.64c1.08 5.54 4.15 9.45 12.41 14.22 16.43 9.49 42.94 9.49 59.24 0 8.11-4.69 10.85-9.74 12.67-15.31z"
            data-name="Pot"
            id="prefix__Pot-2"
          />
          <path
            class="prefix__cls-6"
            d="M382.8 591.26c16.56 9.57 16.63 25.06.19 34.61s-43.16 9.56-59.73 0-16.63-25-.2-34.58 43.19-9.58 59.74-.03z"
            data-name="Pot Top"
            id="prefix__Pot_Top"
          />
          <path
            class="prefix__cls-5"
            d="M380 593.83c15 8.15 15.09 21.34.17 29.47s-39.16 8.14-54.19 0-15.09-21.31-.18-29.45 39.2-8.15 54.2-.02z"
            data-name="Pot Interior"
            id="prefix__Pot_Interior"
          />
          <path
            class="prefix__cls-7"
            d="M379.57 605.08c4.54 2.46 8.53 5.82 10.37 9.18 0 0-1.95 5-10.4 9.41-14.54 7.6-38.27 7.57-52.91-.06-7.36-3.83-9.71-8.11-9.71-8.11 1.45-3.62 5.15-7.59 10.31-10.41 14.41-7.85 37.84-7.86 52.34-.01z"
            id="prefix__Dirt"
          />
        </g>
        <g
          id="prefix__Plant"
        >
          <path
            d="M354.54 505.61a26.63 26.63 0 017.52-3.62l-9 12.83"
            fill="none"
            stroke="#0041f4"
            stroke-miterlimit="10"
            stroke-width="0.26"
          />
          <path
            class="prefix__cls-5"
            d="M340.75 618.15l8.67-37.6s-.14-11.62-5.69-27.17c-7.11-19-18-25.79-18-25.79l13.46 5.41 6-14.63s-2.13-15.58-4.52-22.42a44.65 44.65 0 00-7.54-12.74s4.79.77 10.5 9.71a44.65 44.65 0 016.54 17.83 17.49 17.49 0 014.36-5.15 26.63 26.63 0 017.52-3.62 96.36 96.36 0 00-9 12.83c-4.11 7.12-6 25.23-6 25.23l12 22.45a63.31 63.31 0 0112.07-13.91c7.16-5.9 16.66-8.68 16.66-8.68a98.9 98.9 0 00-12 16.69 79.75 79.75 0 00-8.18 22.45c-1.43 6.39-5.39 38.82-5.39 38.82s-2.25 3-11.18 5.23-10.28-4.94-10.28-4.94z"
            id="prefix__Stem"
          />
          <path
            class="prefix__cls-4"
            d="M340.75 618.15l8.67-37.6s-.14-11.62-5.69-27.17c-7.11-19-18-25.79-18-25.79l13.46 5.41 6-14.63s-2.13-15.58-4.52-22.42a44.65 44.65 0 00-7.54-12.74s5 .69 10.07 10c3.08 5.61 5 22.38 5 22.38a18.52 18.52 0 016.35-9.95 28.87 28.87 0 017.52-3.62s-6.51 2.81-11.65 13.26c-3.62 7.38-7.52 24.8-7.52 24.8s4.62 3.72 8.75 17.79c1.88 6.44 5.17 21.34 3.93 29-1.49 9.16-6.87 26.45-5.52 30.72s4 4.44 4 4.44a9.25 9.25 0 01-3 1.14c-8.88 2.16-10.31-5.02-10.31-5.02z"
            data-name="Stem Shadow"
            id="prefix__Stem_Shadow"
          />
          <path
            class="prefix__cls-4"
            d="M362.42 573.4l-3.53-10.09a44.7 44.7 0 0112.31-14.72c8.55-6.62 15.94-8.45 15.94-8.45a89.62 89.62 0 00-15.72 16.54c-7.35 10.11-9 16.72-9 16.72z"
            data-name="Stem Shadow"
            id="prefix__Stem_Shadow-2"
          />
          <path
            class="prefix__cls-4"
            d="M336.57 483.92S334.3 468.16 322 461c-15.6-9-38.32 3.92-38.32 3.92s9.09 16.75 23 22.64 29.89-3.64 29.89-3.64z"
            id="prefix__Leaf"
          />
          <path
            class="prefix__cls-4"
            d="M329.46 531s-10.55-18.76-29.59-21.72c-24.06-3.69-46.38 23.72-46.38 23.72s19.63 16.67 40 17.3 35.97-19.3 35.97-19.3z"
            data-name="Leaf"
            id="prefix__Leaf-2"
          />
          <path
            class="prefix__cls-4"
            d="M357.56 501.41s2.66-18.46 17.12-26.81c18.27-10.55 44.88 4.59 44.88 4.59s-10.65 19.63-26.93 26.51-35.07-4.29-35.07-4.29z"
            data-name="Leaf"
            id="prefix__Leaf-3"
          />
          <g
            class="prefix__cls-9"
            data-name="Shadow"
            id="prefix__Shadow-2"
          >
            <path
              class="prefix__cls-10"
              d="M336.05 483.23s-8-11.49-17.61-11.89c-8.2-.34-17.22 5.71-21.92 9.39a38.29 38.29 0 0010.42 6.86c12.07 5.1 25.76-2 29.15-3.9-.02-.29-.04-.46-.04-.46z"
            />
          </g>
          <g
            class="prefix__cls-9"
            data-name="Shadow"
            id="prefix__Shadow-3"
          >
            <path
              class="prefix__cls-10"
              d="M328.37 529.63s-11.46 1.5-17.47 6c-4.94 3.69-9 9-10.22 13.42 14.74-2.71 25.89-14.81 28.56-17.95a3.88 3.88 0 00.22-.48c-.15-.17-1.09-.99-1.09-.99z"
            />
          </g>
          <path
            class="prefix__cls-5"
            d="M336.49 483.79s-10.08-9.68-19.62-8.16c-11.59 1.84-22.26 16.86-22.26 16.86s14.06 8 26.87 6.19c19.2-2.68 15.01-14.89 15.01-14.89z"
            data-name="Leaf"
            id="prefix__Leaf-4"
          />
          <path
            class="prefix__cls-5"
            d="M330.06 530.47s-16.42 2.92-21.67 13.18C302 556.12 309 577 309 577s17.59-7.93 24.85-21.55c10.88-20.45-3.79-24.98-3.79-24.98z"
            data-name="Leaf"
            id="prefix__Leaf-5"
          />
          <path
            class="prefix__cls-5"
            d="M357.4 501.35s12.4-6.47 21.13-2.34c10.61 5 16.65 22.42 16.65 22.42s-15.74 3.75-27.53-1.58c-17.65-7.99-10.25-18.5-10.25-18.5z"
            data-name="Leaf"
            id="prefix__Leaf-6"
          />
          <path
            class="prefix__cls-4"
            d="M385 540s10-14.7 22.28-15.32c14.91-.76 32.14 15.13 32.14 15.13s-15.42 13.64-31.81 14.75C383 556.23 385 540 385 540z"
            data-name="Leaf"
            id="prefix__Leaf-7"
          />
          <path
            class="prefix__cls-5"
            d="M339 487.21s-1.28-9.39 1.73-13a15.81 15.81 0 0115.08-5.48s.08 10.81-4.83 15.72c-6.46 6.55-11.98 2.76-11.98 2.76z"
            data-name="Leaf"
            id="prefix__Leaf-8"
          />
          <path
            class="prefix__cls-5"
            d="M352.36 514.92s10.74 2.41 13.42 7.13a18.39 18.39 0 01-.13 18.65s-11.81-4.28-15.17-11.63c-4.48-9.7 1.88-14.15 1.88-14.15z"
            data-name="Leaf"
            id="prefix__Leaf-9"
          />
          <path
            class="prefix__cls-5"
            d="M383.77 542.61s-2.29-8.15-.08-11.69a14.13 14.13 0 0112.67-6.73s1.42 9.55-2.32 14.52c-4.94 6.55-10.27 3.9-10.27 3.9z"
            data-name="Leaf"
            id="prefix__Leaf-10"
          />
          <path
            class="prefix__cls-5"
            d="M384.22 543.15s14.89-3.24 21 1.12c12.95 9.25 6.13 29.81 6.13 29.81s-12.74-4.83-21.25-12.08c-11.22-9.58-5.88-18.85-5.88-18.85z"
            data-name="Leaf"
            id="prefix__Leaf-11"
          />
        </g>
        <g
          id="prefix__Door"
        >
          <path
            d="M839.55 106.49v480.97l241.97 139.72.01-480.69-241.98-140z"
            fill="#bae6ff"
          />
          <path
            class="prefix__cls-7"
            d="M868.84 148.49v455.88l191.44 110.55.01-458.46-191.45-107.97z"
          />
          <path
            class="prefix__cls-12"
            d="M861.88 144.47l6.96 3.93v455.97l-6.96-4.02V144.47zM1081.53 246.52l6.95-4.27v481.03l-6.95 3.9V246.52z"
          />
          <path
            class="prefix__cls-13"
            d="M839.55 106.49l6.8-3.93 242.13 139.69-6.95 4.27-241.98-140.03z"
          />
        </g>
        <path
          class="prefix__cls-4"
          d="M872.79 490.38l2.39 25.53 5.82-2.59-1.06-26.95-7.15 4.01z"
        />
        <path
          class="prefix__cls-14"
          d="M867.83 184.87l-6 4.14 1.34 25.61 5.62-2.82v-26.4a.65.65 0 00-.96-.53z"
        />
        <path
          class="prefix__cls-4"
          d="M868.84 218.34a4.24 4.24 0 01-6.93.64l-2.38-25.79a4.24 4.24 0 006.93-.64z"
        />
        <ellipse
          class="prefix__cls-6"
          cx="862.92"
          cy="192.11"
          rx="3.27"
          ry="2.37"
          transform="rotate(-5.27 862.804 192.132)"
        />
        <path
          class="prefix__cls-4"
          d="M874.77 516.17a4.24 4.24 0 01-6.93.66l-2.48-25.78a4.24 4.24 0 006.92-.67z"
        />
        <ellipse
          class="prefix__cls-6"
          cx="868.75"
          cy="489.96"
          rx="3.27"
          ry="2.37"
          transform="rotate(-5.51 868.129 489.705)"
        />
        <circle
          class="prefix__cls-15"
          cx="879.73"
          cy="497.54"
          r="1.68"
        />
        <circle
          class="prefix__cls-15"
          cx="880.31"
          cy="505.36"
          r="1.68"
        />
        <path
          class="prefix__cls-13"
          d="M868.84 153.94l14.61 436.2 165.03 124.78-15.52-442.38-164.12-118.6z"
        />
        <path
          class="prefix__cls-16"
          d="M868.84 153.94l4.77-2.76 19.64 11.08 147.09 106.59-7.38 3.69-164.12-118.6z"
        />
        <path
          class="prefix__cls-10"
          d="M1032.96 272.54l7.38-3.69 15.86 438.48-7.72 7.59-15.52-442.38z"
        />
        <path
          class="prefix__cls-14"
          d="M859.17 193.15L848.53 208a2 2 0 00-.4 1.73l3.78 16.16a2 2 0 003.57.77l6.19-7.75z"
        />
        <path
          class="prefix__cls-17"
          d="M861.88 600.35l21.57-10.21 165.03 124.78-1.92 1.48-184.68-116.05zM861.88 162.51l6.96-4.01 6.2 181.59-13.16-177.58z"
        />
        <circle
          class="prefix__cls-15"
          cx="854.56"
          cy="209.77"
          r="1.68"
        />
        <circle
          class="prefix__cls-15"
          cx="855.96"
          cy="216.88"
          r="1.68"
        />
        <path
          class="prefix__cls-4"
          d="M863.25 638.66l7.26-3.9a2.59 2.59 0 012.86 1.13 2.74 2.74 0 01.09 3.13l-7.17 4z"
        />
        <path
          class="prefix__cls-15"
          d="M873.73 636.55a2.76 2.76 0 01-.24 2.46l-7.12 4.13-1.25-1.68 2.38-1.4.13-.58.65.14.12-.58.65.13.13-.57.65.14.13-.57.65.13.13-.57.65.13.13-.57.65.13.13-.57.65.13.13-.57z"
        />
        <path
          class="prefix__cls-15"
          d="M866.89 639.7c1.47 2.68 1 5.82-1.11 7.24-.11.07-.59.43-.71.49-2.35 1.29-5.15-.23-6.7-3.06-1.46-2.66-1.93-4.38-.24-6.21a8.49 8.49 0 011.48-1.12l.22-.13c2.35-1.29 5.52-.03 7.06 2.79z"
        />
        <ellipse
          class="prefix__cls-4"
          cx="861.82"
          cy="642.61"
          rx="4.86"
          ry="5.83"
          transform="rotate(-28.73 861.778 642.606)"
        />
        <path
          class="prefix__cls-15"
          d="M859.23 639.31a.61.61 0 01.83.2l4 6.25a.63.63 0 01-.16.85.63.63 0 01-.89-.17l-4-6.24a.63.63 0 01.22-.89z"
        />
        <path
          class="prefix__cls-15"
          d="M864.49 640.68a.63.63 0 01-.1.86l-4.47 3.6a.62.62 0 01-.85 0 .63.63 0 010-.91l4.45-3.63a.62.62 0 01.97.08z"
        />
        <path
          class="prefix__cls-4"
          d="M782.76 584.49l7.28-3.88a2.61 2.61 0 012.86 1.14 2.75 2.75 0 01.07 3.14l-7.18 4z"
        />
        <path
          class="prefix__cls-15"
          d="M793.25 582.42a2.74 2.74 0 01-.25 2.46l-7.14 4.12-1.24-1.68 2.38-1.42.13-.58.65.14.12-.57.65.13.14-.57.65.14.13-.57.65.13.13-.57.65.14.13-.57.65.13.13-.57.65.14.13-.57z"
        />
        <path
          class="prefix__cls-15"
          d="M786.4 585.54c1.46 2.69.93 5.83-1.14 7.23-.11.08-.59.43-.71.49-2.36 1.28-5.15-.26-6.69-3.08s-1.91-4.4-.21-6.22a8.27 8.27 0 011.48-1.11l.22-.13c2.36-1.28 5.52-.01 7.05 2.82z"
        />
        <ellipse
          class="prefix__cls-4"
          cx="781.31"
          cy="588.43"
          rx="4.86"
          ry="5.83"
          transform="rotate(-28.49 781.18 588.38)"
        />
        <path
          class="prefix__cls-15"
          d="M778.74 585.12a.62.62 0 01.83.21l4 6.27a.62.62 0 01-.16.84.62.62 0 01-.89-.17l-4-6.25a.62.62 0 01.22-.9z"
        />
        <path
          class="prefix__cls-15"
          d="M784 586.51a.61.61 0 01-.11.85l-4.46 3.64a.63.63 0 01-.86-.06.63.63 0 01.06-.9l4.47-3.62a.62.62 0 01.9.09z"
        />
        <path
          class="prefix__cls-4"
          d="M817.88 639.47l-7-4.34a2.62 2.62 0 01-.45-3 2.74 2.74 0 012.67-1.64l7.08 4.2z"
        />
        <path
          class="prefix__cls-15"
          d="M810.82 631.44a2.77 2.77 0 012.26-1l7.13 4.12-.84 1.92-2.37-1.38-.56.17-.21-.63-.56.18-.21-.63-.56.17-.21-.63-.56.17-.21-.63-.55.17-.21-.63-.56.18-.21-.63-.56.17-.21-.63-.56.17z"
        />
        <path
          class="prefix__cls-15"
          d="M817.39 635.8c1.84-2.44 4.92-3.27 7.06-2 .11.07.63.35.74.44 2.14 1.61 1.9 4.79 0 7.36s-3.2 3.57-5.56 2.78a8.36 8.36 0 01-1.63-.89l-.21-.14c-2.17-1.57-2.33-4.98-.4-7.55z"
        />
        <ellipse
          class="prefix__cls-4"
          cx="822.14"
          cy="639.23"
          rx="5.83"
          ry="4.86"
          transform="rotate(-53.05 822.131 639.225)"
        />
        <path
          class="prefix__cls-15"
          d="M821 643.58a.62.62 0 01-.4-.76l2.08-7.13a.62.62 0 01.74-.43.63.63 0 01.46.78l-2 7.14a.61.61 0 01-.88.4z"
        />
        <path
          class="prefix__cls-15"
          d="M818.6 638.72a.63.63 0 01.71-.49l5.66 1a.65.65 0 01.52.69.64.64 0 01-.72.55l-5.67-1a.63.63 0 01-.5-.75zM787.93 609.74c-2.92-2.43-5.93-1.69-7.9.68s-1.88 5.71 1 8.14l.71.57c2.92 2.39 6.86 2.43 8.82.06s1.22-6.15-1.6-8.58zm.56 7.7c-1.18 1.42-3.56 1.4-5.32-.06s-2.21-3.79-1-5.21 3.56-1.4 5.32.06 2.18 3.77 1 5.21z"
        />
        <path
          class="prefix__cls-4"
          d="M787.93 609.74c-2.92-2.43-6.89-2.47-8.85-.1s-1.2 6.26 1.71 8.68 6.88 2.47 8.85.1 1.2-6.26-1.71-8.68zm-.4 6.92c-1.18 1.43-3.56 1.4-5.31 0s-2.22-3.79-1-5.22 3.56-1.4 5.31.06 2.19 3.74 1 5.16z"
        />
        <path
          class="prefix__cls-15"
          d="M853.36 617c-2.92-2.43-5.93-1.69-7.9.68s-1.88 5.71 1 8.14l.72.57c2.91 2.39 6.85 2.43 8.81.06s1.22-6.15-1.6-8.58zm.56 7.7c-1.18 1.42-3.56 1.4-5.31-.06s-2.22-3.79-1-5.21 3.57-1.4 5.32.06 2.17 3.81.99 5.23z"
        />
        <path
          class="prefix__cls-4"
          d="M853.36 617c-2.92-2.43-6.88-2.47-8.85-.1s-1.2 6.26 1.72 8.68 6.88 2.47 8.85.1 1.2-6.24-1.72-8.68zm-.4 6.92c-1.18 1.43-3.56 1.4-5.31-.05s-2.21-3.79-1-5.22 3.56-1.39 5.31.06 2.19 3.81 1.04 5.23z"
        />
        <path
          class="prefix__cls-15"
          d="M855 213.46a4 4 0 015.21.92c.05.08.3.43.35.52.89 1.71-.25 3.71-2.31 4.79s-3.2 1.33-4.5.08a6 6 0 01-.78-1.09l-.09-.15c-.88-1.72.03-3.99 2.12-5.07z"
        />
        <ellipse
          class="prefix__cls-4"
          cx="857"
          cy="217.18"
          rx="4.21"
          ry="3.51"
          transform="rotate(-27.53 856.986 217.167)"
        />
        <path
          class="prefix__cls-15"
          d="M854.92 219.67a.45.45 0 010-.62l3.57-4a.45.45 0 01.62-.05.45.45 0 01.06.65l-3.56 4a.45.45 0 01-.69.02z"
        />
        <path
          class="prefix__cls-15"
          d="M854.85 215.75a.45.45 0 01.62-.1l3.38 2.41a.44.44 0 01.13.6.46.46 0 01-.64.14l-3.4-2.39a.44.44 0 01-.09-.66z"
        />
        <path
          class="prefix__cls-7"
          d="M995.44 455.66l2.37 59a.91.91 0 00.56.82l21.86 11.22a1.11 1.11 0 001 0c.37-.19 2.64-2.87 2.62-3.34l-2.3-57.16a2.62 2.62 0 00-1.55-2.2l-20.83-11c-.79-.35-3.19 1.53-3.51 1.89a1 1 0 00-.22.77z"
        />
        <path
          class="prefix__cls-6"
          d="M995.39 455.58l2.37 59a1.09 1.09 0 00.59.94l21.89 11.22a1.1 1.1 0 001.62-1l-2.33-58a2.76 2.76 0 00-1.48-2.35l-20.68-10.9a1.45 1.45 0 00-1.98 1.09z"
        />
        <path
          class="prefix__cls-10"
          d="M1003.49 491.46a5.57 5.57 0 016.09-4.9 5.34 5.34 0 00-1-.21 5.57 5.57 0 10-1.28 11.07h.08a5.56 5.56 0 01-3.89-5.96z"
        />
        <path
          class="prefix__cls-7"
          d="M1009.56 486.55a5.58 5.58 0 103.93 6 5.58 5.58 0 00-3.93-6z"
        />
        <g
          class="prefix__cls-9"
        >
          <ellipse
            class="prefix__cls-7"
            cx="981.96"
            cy="722.79"
            rx="9.16"
            ry="0.97"
          />
        </g>
        <g
          data-name="Door Knob"
          id="prefix__Door_Knob"
        >
          <path
            class="prefix__cls-10"
            d="M989.93 714.76a7.9 7.9 0 01-1.08 4.34 7.5 7.5 0 01-.61.9 8 8 0 11-10-12 8.41 8.41 0 011.71-.66 7.73 7.73 0 011.67-.24 8 8 0 018.31 7.66z"
          />
          <circle
            class="prefix__cls-4"
            cx="980.56"
            cy="714.95"
            r="8.16"
          />
          <circle
            class="prefix__cls-16"
            cx="980.4"
            cy="714.99"
            r="7.5"
          />
          <circle
            class="prefix__cls-4"
            cx="979.88"
            cy="715.59"
            r="7.2"
          />
        </g>
        <g
          id="prefix__Sign"
        >
          <path
            class="prefix__cls-5"
            d="M899.28 54.18v63.01l122.94 70.93-.04-62.94-122.9-71z"
          />
          <path
            class="prefix__cls-7"
            d="M899.28 54.18l6.02-3.57L1028.47 122l-6.29 3.18-122.9-71z"
          />
          <path
            class="prefix__cls-10"
            d="M1022.18 125.18l6.29-3.18v60.93l-6.25 5.19-.04-62.94z"
          />
          <path
            class="prefix__cls-16"
            d="M904.13 63.13v50.94l112.99 65.22-.03-51.14-112.96-65.02z"
          />
        </g>
        <path
          class="prefix__cls-10"
          d="M955.86 99.31l-.02 25.13 7.19 4.08.05-25.04-7.22-4.17zM955.84 128.97v8.33l7.24 4.19v-8.35l-7.24-4.17z"
        />
      </svg>
      <div
        class="cds--bmrg-error-page-core__content"
      >
        <h1
          class="cds--bmrg-error-page-core__header"
        >
          Oops!
        </h1>
        <p
          class="cds--bmrg-error-page-core__title"
        >
          Something looks off, but we're getting a handle of it.
        </p>
        <section
          class="cds--bmrg-error-page-core__message"
        >
          <p>
            Hit the back button to return to your previous page, or if you keep finding yourself here,
            <a
              href="https://useboomerang.io"
            >
               navigate to Support Center 
            </a>
            for help.
          </p>
        </section>
      </div>
    </div>
  </div>
</body>
`;
